src/aws-lambda/retaildemostore-lambda-load-products/src/main.go

```diff
@@ -161,6 +162,8 @@ func loadData(s3bucket, s3file, ddbtable, datatype string) (string, error) {
 // HandleRequest - handles Lambda request
 func HandleRequest(ctx context.Context, event cfn.Event) (physicalResourceID string, data map[string]interface{}, err error) {
 
+    fmt.Sprintf("Received event: %s!", event )
```

Comment:
This does nothing with the resulting string. Remove or do something with the string.